jsonschema-validation.xml

```diff
@@ -297,31 +297,59 @@
                 </t>
             </section>
 
-            <section title="additionalItems and items">
+            <section title="items">
                 <t>
-                    The value of "additionalItems" MUST be either a boolean or an object. If
-                    it is an object, this object MUST be a valid JSON Schema.
+                    The value of "items" MUST be either an object or an array of objects.
+                    Each object MUST be a valid JSON Schema.
                 </t>
                 <t>
-                    The value of "items" MUST be either a schema or array of schemas.
+                    If absent, it can be considered present with an empty schema.
                 </t>
                 <t>
-                    Successful validation of an array instance with regards to these two
-                    keywords is determined as follows:
+                    Validation of this keyword against the instance always succeeds.
```

Comment:
That's because it's weird :-P

While working out schema algebra rules for the whole `"additionalProperties": false"` thing, it became clear that it's very useful to have separate notions of whether the immediate instance validates against the keyword and whether any children validate.

For an array, immediate instance validation is just about about the array length.

So in this case, the "items" keyword cannot place restrictions on the length, so the array (as an array, independent of its elements) cannot fail validation based on this keyword.  Even if "items" is itself an array, it does not on its own restrict the length of the array- that is only done by "minItems" and "maxItems".

Even `"additionalItems": false` does not impose length restrictions, it just ensures that child validation fails for all elements after a certain point.  Actually, I think the wording can be improved around that now that I've tried to explain it.